client/src/featureform/dashboard_metadata.py

```diff
@@ -119,6 +121,8 @@ def training_set_variant(variantData):
                 "Variant":variantRow['label_variant']}, 
                 variantRow['status'], 
                 variant_organiser(feature_variant(getTrainingSetFeatures(feature_list))[2]),
+                json.loads(variantRow['tags']) if variantRow['tags'] is not None else [],
```

Comment:
Why are we storing this as JSON? Where is it JSON?

Reply:
I figured it would simpler to store JSON rather that handling multiple rows (e.g. a record per tag). But honestly, it was an arbitrary decision.